presto-cli/src/main/java/com/facebook/presto/cli/Console.java

```diff
@@ -322,11 +324,41 @@ private static void process(QueryRunner queryRunner, String sql, OutputFormat ou
             queryRunner.setSession(session);
         }
         catch (RuntimeException e) {
-            System.err.println("Error running command: " + e.getMessage());
-            if (queryRunner.getSession().isDebug()) {
-                e.printStackTrace();
+            System.err.println(getErrorMessage(e, queryRunner.getSession()));
+        }
+    }
+
+    private static String getErrorMessage(RuntimeException e, ClientSession session)
+    {
+        StringWriter sw = new StringWriter();
+        e.printStackTrace(new PrintWriter(sw));
+        String stackTrace = sw.toString();
```

Comment:
I would wrapt this as static method utility method `getStackTrace(e)`, moreove this variable is used in condition block so I would create this variable there